src/lib.rs

```diff
@@ -188,6 +188,15 @@ impl Builder {
         self
     }
 
+    /// Whether to whitelist types recursively or not. Defaults to true.
+    ///
+    /// See https://github.com/servo/rust-bindgen/issues/429 for different use
+    /// cases for this.
```

Comment:
I think we should elaborate the use cases here: one shouldn't have to jump from the rustdocs to a random github issue to understand why a method exists and to figure out if they should use it or not.